libs/shared/ui-components/src/SEO/SEO.tsx

```diff
@@ -1,14 +1,63 @@
 import { FC } from 'react';
 
 import Head from 'next/head';
+import { useRouter } from 'next/router';
 
 import { DomainVariant } from '@quansight/shared/types';
 
 import { TSEOProps } from './types';
 
-export const SEO: FC<TSEOProps> = ({ title, description, variant }) => (
-  <Head>
-    <title>{`${title} | ${DomainVariant[variant]}`}</title>
-    <meta name="description" content={description} />
-  </Head>
-);
+export const SEO: FC<TSEOProps> = ({ title, description, variant }) => {
+  const { asPath } = useRouter();
+  const fallBackOrigin =
+    DomainVariant[variant] === 'Quansight'
+      ? 'quansight.com'
```

Comment:
I think we need this to be absolute

```suggestion
      ? 'https://quansight.com'
```